crates/rolldown_plugin_transform/src/lib.rs

```diff
@@ -40,7 +40,7 @@ pub struct TransformPlugin {
 /// only handle ecma like syntax, `jsx`,`tsx`,`ts`
 impl Plugin for TransformPlugin {
   fn name(&self) -> Cow<'static, str> {
-    Cow::Borrowed("module_preload_polyfill")
+    Cow::Borrowed("transform")
```

Comment:
We should have a convention for this. `builtin:transform` would be more proper I guess.

@underfin cc

Reply:
All plugins don't have a namespace for now, prefix a namespace is better.